src/pinecone-generated-ts-fetch/control/models/IndexModelStatus.ts

```diff
@@ -45,6 +45,7 @@ export const IndexModelStatusStateEnum = {
   ScalingDownPodSize: 'ScalingDownPodSize',
   Terminating: 'Terminating',
   Ready: 'Ready',
+  Upgrading: 'Upgrading',
```

Comment:
see new `Bug fixes` section in PR description for more info

Reply:
This model is from generated code, so we shouldn't be updating it manually. This corresponds to this in the spec: https://github.com/pinecone-io/apis/blob/0098ac4ff262286f371bb8b6803549a3446e5065/src/release/db/control/resources/indexes/models/IndexModel.yaml#L58-L66

I don't think we actually represent it in the spec at all, but it should probably be fixed upstream if we need to.